inst/specs/ames.yml

```diff
@@ -0,0 +1,623 @@
+    values:
+      Floor: forno de piso
+      GasA: forno de ar quente a gás
+      GasW: água quente a gás ou calor de vapor
+      Grav: forno de gravidade
+      OthW: água quente ou calor de vapor diferente de gás
+      Wall: forno de parede
+  Heating QC:
+    trans: aquecimento_qualidade_condicao
+    desc: Avaliação da qualidade e condição do aquecimento
+    values:
+      Ex: excelente
+      Fa: regular
+      Gd: boa
+      Po: ruim
+      TA: média
+  Central Air:
+    trans: ar_condicionado_central
+    desc: Ar condicionado central da casa
+    values:
+      N: não
+      Y: sim
+  Electrical:
+    trans: sistema_eletrico_tipo
+    desc: Tipo de sistema elétrico da casa
+    values:
+      FuseA: caixa de fusíveis acima de 60 AMP e toda a fiação Romex (média)
+      FuseF: caixa de fusíveis de 60 AMP e principalmente fiação Romex (regular)
+      FuseP: caixa de fusível de 60 AMP e principalmente fiação de botão e tubo (ruim)
+      Mix: misto
+      SBrkr: disjuntores padrão e Romex
+  1st Flr SF:
+    trans: primeiro_andar_area
+    desc: Área do primeiro andar (pés quadrados)
+  2nd Flr SF:
+    trans: segundo_andar_area
+    desc: Área do segundo andar (pés quadrados)
+  Low Qual Fin SF:
+    trans: acabamento_baixa_qualidade_area
+    desc: Área da casa com acabamento de baixa qualidade (pés quadrados)
+  Gr Liv Area:
+    trans: acima_solo_area
+    desc: Área da casa acima do solo (pés quadrados)
+  Full Bath:
+    trans: acima_solo_num_banheiros
+    desc: Número de banheiros acima do solo
+  Half Bath:
+    trans: acima_solo_num_lavabos
+    desc: Número de lavabos acima do solo
+  Bedroom AbvGr:
+    trans: acima_solo_num_quartos
+    desc: Número de quartos da casa
+  Kitchen AbvGr:
+    trans: acima_solo_num_cozinhas
+    desc: Número de cozinhas da casa
+  Kitchen Qual:
+    trans: cozinha_qualidade
+    desc: Qualidade da cozinha
+    values:
+      Ex: excelente
+      Fa: regular
+      Gd: boa
+      Po: ruim
+      TA: média
+  TotRms AbvGrd:
+    trans: acima_solo_num_comodos
+    desc: Número de cômodos acima do solo
+  Functional:
+    trans: funcional
+    desc: Funcionalidade da casa (indica o quanto a casa está apta para moradia)
+    values:
+      Maj1: deduções maiores 1
+      Maj2: deduções maiores 2
+      Min1: deduções menores 1
+      Min2: deduções menores 2
+      Mod: deduções moderadas
+      Sal: apenas recuperação
+      Sev: severamente danificada
+      Typ: funcionalidade típica
+  Fireplaces:
+    trans: total_num_lareiras
+    desc: Número total de lareiras
+  Fireplace Qu:
+    trans: lareira_qualidade
+    desc: Qualidade da(s) lareira(s)
+    values:
+      Ex: excelente (fogueira de alvenaria exepecional)
+      Fa: regular (fogueira pré-fabricada no porão)
+      Gd: boa (lareira de alvenaria no nível principal)
+      Po: ruim (fogão Ben Franklin)
+      TA: média (lareira pré-fabricada na sala de estar principal ou lareira de alvenaria no porão)
```

Comment:
@williamorim @beatrizmilz 

I just realized that in many cases you made the translated values extreamly long. I'm not sure if this is always useful. If you plot this variables you'll have to edit the way the values appear in the axes. And if your screen is small you can't see the complete values in the Console. Maybe is a better idea to keep the values short (for example,  `boa` instead of `boa (lareira de alvenaria no nível principal)`), and include a link to a translated version of the data documentation.

![Rplot01](https://user-images.githubusercontent.com/31421616/94627508-268b2280-0294-11eb-9683-fdb40b135072.png)

![Captura de pantalla de 2020-09-29 20-44-03](https://user-images.githubusercontent.com/31421616/94627673-90a3c780-0294-11eb-9501-8df3874d0845.png)

Reply:
Or we can add this translation to the description of the variable, to appear in the documentation. Ex:

```
lareira_qualidade
Qualidade da(s) lareira(s). Categorias: excelente (fogueira de alvenaria excepcional); boa (lareira de alvenaria no nível principal),  média (lareira pré-fabricada na sala de estar principal ou lareira de alvenaria no porão),  regular (fogueira pré-fabricada no porão), ruim (fogão Ben Franklin).
```

What do you think @rivaquiroga and @williamorim ?